README.md

```diff
@@ -42,8 +42,14 @@ var foods = [ ["apple","banana","strawberry"], ["pizza","fries","hamburger"] ];
 Create a For Loop that outputs the following string for each piece of fruit in the console. "I want to eat a [fruit]"
 
 ```js
-// write code here
+for (i=0; i<foods.length; i++){
+    for (x=0; x<foods[i].length; x++)
+
+    console.log("I want to eat a " + foods[i][x]);
+}
```

Comment:
Close.  Can we limit it to just the fruits?  Also, your second loop was not closed (no closing "}").